store/notifyer.go

```diff
@@ -0,0 +1,71 @@
+package store
```

Comment:
Since this feature is specific to store.FileStore and not agnostic to all store implementations, please move this code inside the store.FileStore file.

Reply:
Moved to `store.FileStore`